mainwindow.ui

```diff
@@ -66,9 +66,6 @@
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
-      <property name="sizeAdjustPolicy">
```

Comment:
This property don't exist on old qt version and we can't put #define in the .ui file so we get a bug where the tab menu takes like 2-3 times more space than needed but the text of the option stay as small with Mot...XXX . Maybe we should reput the workaround that Aleksey did in 2012 that we removed at the beggining of the summer. When we added adjustToContents property, it didn't actually solved the ... bug that we had with translation english to french and incomplete word with ... for the QLIstWidget